superset-frontend/src/dashboard/components/gridComponents/ChartHolder.tsx

```diff
@@ -76,6 +76,7 @@ const fullSizeStyle = css`
     z-index: 3000;
     left: 0;
     top: 0;
+    padding: ${theme.gridUnit * 2}px;
```

Comment:
Oops... just need to add a `const theme = useTheme();`  before the `const fullSizeStyle...` bit, and pull in `{ useTheme } from '@superset-ui/core'`